contracts/NestingMultiResource/README.md

```diff
@@ -45,14 +45,28 @@ The `constructor` in this case accepts no arguments as all of the arguments requ
 - `1000`: represents the `maxSupply_` argument and sets the maximum amount of tokens in the collection
 - `100_000_000`: represents the `pricePerMint_` argument and sets the price of minting one token in wei or the lowest
 denomination of the native currency of the EVM to which the smart contract is deployed to
-- `ipfs://meta`: represents the `collectionMetadata_` argument and sets the base URI of the collection metadata
+- `ipfs://meta`: represents the `collectionMetadata_` argument and sets the URI of the collection metadata
+- `ipfs://tokenMeta`: represents the `tokenURI_` argument and sets the base URI of the token metadata
+- `msg.sender`: represents the `royaltyRecipient` argument and sets the address of the recipient of royalty fees
+- `10`: represents the `royaltyPercentageBps` argument and sets the value of royalty percentage in basis points
```

Comment:
I would like to use a more real value here so it does not confuse users. For instance `500` and mention that it is equivalent to 5%

Reply:
Of course, I can use such value in all of the instances, or rather add it. I'd like to keep the lowest possible value mentioned here as well 🤔